fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/SoundPlayer.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.item.v1;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.world.World;
+
+@FunctionalInterface
+public interface SoundPlayer {
+	void playSound(World world, PlayerEntity player);
```

Comment:
Where is this called? Is this called on the logical client, logical server or both?

Reply:
It's called on both